master/internal/api_experiment.go

```diff
@@ -23,6 +23,31 @@ import (
 	"github.com/determined-ai/determined/proto/pkg/experimentv1"
 )
 
+func isInList(srcList []string, item string) bool {
+	item = strings.ToLower(item)
+	for _, src := range srcList {
+		if matchesSrc := strings.Contains(strings.ToLower(src), item); matchesSrc {
+			return true
+		}
+	}
+	return false
+}
+
+// matchesList checks whether srcList contains all strings provided in matchList.
+func matchesList(srcList []string, matchList []string) bool {
+	if len(matchList) != 0 && len(srcList) == 0 {
+		return false
+	}
```

Comment:
Is this clause necessary?

Reply:
you're right. nope we don't need it. :rocket: